crates/bevy_animation/src/lib.rs

```diff
@@ -462,7 +462,7 @@ fn verify_no_ancestor_player(
         let Ok((maybe_player, parent)) = parents.get(current) else {
             return true;
         };
-        if maybe_player.is_some() {
+        if maybe_player {
```

Comment:
Nit: Maybe a rename can make sense here
```suggestion
        if has_animation_player {
```